src/index.html

```diff
@@ -895,6 +896,22 @@ <h5>Archive process</h5>
                                         </div>
                                     </li>
                                 </div>
+                                <div id="what-data-is-brokered-to-dbsnp">
+                                    <li class="accordion-item is-active" data-accordion-item>
+                                        <a class="accordion-title"><h5>Which data is brokered to DBSNP?</h5></a>
+                                        <div class="accordion-content" data-tab-content>
+                                            <p>Any variation dataset containing human genotype or allele frequency will be brokered to dbSNP for variant accessioning an assignment of RS ids.</p>
+                                        </div>
+                                    </li>
+                                </div>
+                                <div id="how-do-we-deal-with-chromosome-name-diversity">
+                                    <li class="accordion-item is-active" data-accordion-item>
+                                        <a class="accordion-title"><h5>How do we deal with Chromosome name diversity?</h5></a>
+                                        <div class="accordion-content" data-tab-content>
+                                            <p>Contig naming convention can vary depending on how the reference genome was obtained. We strive to support multiple chromosome name such as INSDC or Refseq identifiers and ENA and Genbank chromosome names. In case we do not support the naming convention you have used please contact eva-helpdesk@ebi.ac.uk.</p>

```

Comment:
```suggestion
                                            <p>Contig naming convention can vary depending on how the reference genome was obtained. We strive to support multiple chromosome names, such as INSDC or RefSeq identifiers and ENA and Genbank chromosome names. In case we do not support the naming convention you have used, please contact <a href="mailto:eva-helpdesk@ebi.ac.uk">eva-helpdesk@ebi.ac.uk</a>.</p>

```

Reply:
updated